rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
@nategraf yeah, agree with you but I said maybe we need test a case like Thomas met, which has a defined FQDN.